lib/scala/kg/src/main/scala/stores/kg/KgNodeField.scala

```diff
@@ -0,0 +1,22 @@
+package stores.kg
+
+import models.kg.KgNode
+
+object KgNodeField extends Enumeration {
+  private val ListDelimChar = '|'
+  private val ListDelimString = ListDelimChar.toString
+
+  type KgNodeField = Value
+  val Id = Value("id")
+  val Labels = Value("labels")
+  val Sources = Value("sources")
+  val PageRank = Value("pageRank")
+
+  def getFieldString(node: KgNode, field: KgNodeField): String =
```

Comment:
This is awkward. Put it on KgNode, getFieldString(field: KgNodeField).

Reply:
Your assumption here appears to be incorrect, in any case. Two nodes may have the same set of labels in different order?

Reply:
Gotcha, added `getFieldString(field: KgNodeSortableField)` to `KgNode`

Reply:
And sorted `label` and `sources` before `mkString`. Forgot about the set mechanics